com.unity.multiplayer.mlapi/Tests/Runtime/TickSystemTests.cs

```diff
@@ -6,10 +6,10 @@
 
 namespace MLAPI.RuntimeTests
 {
-    public class TickSystemTests: IDisposable
+    public class TickSystemTests : IDisposable
     {
         private NetworkTickSystem m_TickSystem = null;
-        private float m_TestDuration = 5.0f;
+        private float m_TestDuration = 3.0f;
```

Comment:
Nit, but I would probably make the test duration be expressed as some multiple of the tick interval (e.g. 100 * m_TickInterval), and by the same token make the sleep interval expressed as some fraction of the tick interval vs. having purely magic number constants

Reply:
agreed — @jeffreyrainy for more visibility on your feedback :)